### PR TITLE
aruha-1636: unlimited retention time by Nakadi admin

### PR DESCRIPTION
**Pull request by adyach**
# One-line summary

> Zalando ticket : aruha-1636 (only if appropriate)

## Description
Allow Nakadi admin set unlimited retention time

### CODE REVIEW

File: src/test/java/org/zalando/nakadi/validation/EventTypeOptionsValidatorTest.java [hunk 1]
```diff
@@ -1,51 +1,48 @@
...
-    }
 
     @Test
     public void testValidation() {
         final EventTypeOptions eventTypeOptions = new EventTypeOptions();
-        validator.validate(eventTypeOptions, mockedErrors);
-
-        Mockito.verify(mockedErrors, Mockito.times(0)).rejectValue(Matchers.any(), Matchers.any(),
-                Matchers.any());
+        validator.checkRetentionTime(eventTypeOptions);
```
[the remark this answers is not in the document]
Comment (adyach) [reply src/test/java/org/zalando/nakadi/validation/EventTypeOptionsValidatorTest.java]: 12829615

**Comment by rcillo** [comment]
👍

**Comment by adyach** [comment]
deploy validation please

**Comment by adyach** [comment]
deploy validation please

**Comment by adyach** [comment]
👍

**Pull request merged**